src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -6,13 +6,14 @@
 
 import org.apache.log4j.BasicConfigurator;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Isolation;
 import org.springframework.transaction.annotation.Transactional;
 
 import com.revature.domain.Batch;
 import com.revature.domain.Person;
-import com.revature.domain.PersonBatchJoin;
 import com.revature.domain.PersonRole;
 import com.revature.log.IntEvalLogger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.revature.log.IntEvalLogger'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)